CONTRIBUTING.md

```diff
@@ -0,0 +1,192 @@
+Contributing to `glue` (Adapted from [`astropy`](https://github.com/astropy/astropy))
+================================================
```

Comment:
I think the Adapted from astropy bit could go in a footer at the bottom, it doesn't need to be in the title